docs/source/api/data-management/accessor/csr.rst

```diff
@@ -0,0 +1,43 @@
+.. ******************************************************************************
+.. * Copyright 2023 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. highlight:: cpp
+.. default-domain:: cpp
+
+.. _api_csr_accessor:
+
+=====================================
+Compressed Sparse Rows (CSR) accessor
+=====================================
+
+The ``csr_accessor`` class provides a read-only access to the rows of the
+:txtref:`csr_table` as data arrays in :capterm:`CSR <CSR data>` storage format.
+
+-------------
+Usage example
```

Comment:
```suggestion
Example
```

Reply:
This is a common chapter's name in oneDAL docs. Should I change it everywhere?

Reply:
@Vika-F in that case let's keep it but capitalize both words - Usage Example

Reply:
@aepanchi Then I need to change all 23 files that contain "Usage example" naming anyway. Will change to "Example" then.

Reply:
@aepanchi Oh no.
In the algorithms' docs we have the parts that look like:

```
-------------
Usage Example
-------------

.. include:: ../../../includes/clustering/dbscan-usage-examples.rst

--------
Examples
--------

.. include:: ../../../includes/clustering/dbscan-examples.rst
```

So, it's better to have "Usage Example", and not change it to "Example".

Reply:
@Vika-F makes sense! I will fix the other instances of the incorrect naming (aka Usage **e**xample)